doc/api/url.md

```diff
@@ -237,6 +237,22 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+`url.resolve` behaves differently based on the protocol of the URL in the
+`from` parameter. Protocols like `http`, `https`, `file`, and `ftp` are treated
+specially to match browser behavior, like so.
```

Comment:
And I'm still confused, ftp(s) strips the hostname if it doesn't have a slash on it, but http(s) does not? And file does... what?